Examples/JamitFoundationExample/App/Sources/TableView/Cells/Item/TableViewItemViewModel.swift

```diff
@@ -0,0 +1,24 @@
+//  Copyright © 2020 Jamit Labs GmbH. All rights reserved.
+
+import JamitFoundation
+import Foundation
+
+struct TableViewItemViewModel: ViewModelProtocol {
+    let imageURL: URL?
+    let description: String?
```

Comment:
I think we should avoid to name a variable `description` which may cause a confusion with `description` of `CustomStringConvertible`.

Reply:
Renamed it to `details`